jcstress-core/src/main/java/org/openjdk/jcstress/JCStress.java

```diff
@@ -208,4 +226,25 @@ public SortedSet<String> getTests() {
         return s;
    }
 
+    public int listTests(Options opts, JCStress jcstress) {
```

Comment:
You don't need to pass `JCStress jcstress` now that the method is instance one.

Reply:
//me facepalm